blog/migrations/0003_alter_commentary_options.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-11-07 16:31
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("blog", "0002_alter_commentary_options_alter_post_options"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="commentary",
+            options={
+                "ordering": ["-created_time"],
+                "verbose_name": "commentary",
+                "verbose_name_plural": "commentarys",
```

Comment:
The `verbose_name_plural` for `Commentary` should be 'commentaries' instead of 'commentarys'. This is a grammatical correction to ensure consistency with standard English pluralization.

Reply:
i want write it like "commentarys", it is my idea